dashboard/config/scripts/allthethings.script

```diff
@@ -116,8 +116,8 @@ stage 'Studio'
 level 'Resizing Test'
 
 stage 'Long assessment'
-assessment 'Test longassessment levelgroup'
-assessment 'Test longassessment levelgroup split'
+assessment 'Test longassessment levelgroup new'
+assessment 'Test longassessment levelgroup split new'
```

Comment:
These have been used by so few people that I'm not sure it's crucial to rename them at this point.